sql/hive/src/test/scala/org/apache/spark/sql/hive/MultiDatabaseSuite.scala

```diff
@@ -269,46 +269,46 @@ class MultiDatabaseSuite extends QueryTest with SQLTestUtils with TestHiveSingle
   test("invalid database name and table names") {
     {
       val message = intercept[AnalysisException] {
-        df.write.format("parquet").saveAsTable("`d:b`.`t:a`")
+        df.write.format("parquet").saveAsTable("`db`.`t:a`")
       }.getMessage
-      assert(message.contains("Database 'd:b' not found"))
+      assert(message.contains("Database 'db' not found"))
```

Comment:
ditto. why need to change these tests?

Reply:
Similar to [#comments](https://github.com/apache/spark/pull/33339#discussion_r698126897)